src/reformatters/u_arizona/swann/region_job.py

```diff
@@ -0,0 +1,94 @@
+from collections.abc import Mapping, Sequence
+from pathlib import Path
+from typing import ClassVar
+
+import numpy as np
+import xarray as xr
+from pydantic import Field
+from rasterio import rasterio  # type: ignore
+
+from reformatters.common.download import http_download_to_disk
+from reformatters.common.region_job import (
+    CoordinateValueOrRange,
+    RegionJob,
+    SourceFileCoord,
+)
+from reformatters.common.types import Array2D, ArrayFloat32, Dim, Timestamp
+
+from .template_config import SWANNDataVar
+
+
+class SWANNSourceFileCoord(SourceFileCoord):
+    possible_data_statuses: ClassVar[tuple[str, ...]] = ("stable", "provisional")
+
+    remaining_data_statuses: list[str] = Field(
+        default_factory=lambda: list(SWANNSourceFileCoord.possible_data_statuses)
+    )
+
+    time: Timestamp
+
+    def get_url(self, status: str = "stable") -> str:
+        water_year = self.get_water_year()
+        year_month_day = self.time.strftime("%Y%m%d")
+
+        try:
+            data_status = self.get_data_status()
+        except IndexError:
+            # Allow get_url calls to return a URL for the last attempted status, even if it failed
+            data_status = SWANNSourceFileCoord.possible_data_statuses[-1]
+
+        return f"https://climate.arizona.edu/data/UA_SWE/DailyData_4km/WY{water_year}/UA_SWE_Depth_4km_v1_{year_month_day}_{data_status}.nc"
+
+    def out_loc(
+        self,
+    ) -> Mapping[Dim, CoordinateValueOrRange]:
+        return self.model_dump(
+            exclude=["status", "downloaded_path", "remaining_data_statuses"]  # type: ignore
+        )
```

Comment:
Lets be less magical :)
```suggestion
        return {"time": self.time}
```